### PR TITLE
Rename Project Package to SellSavvy

### LINKED ISSUE #41: Rename Project Package to SellSavvy

### PR DESCRIPTION
**Pull request by FooChao**
I also changed the logger name from ab3 to sellSavvy
I also edit the javadoc for editPersonCommandParser and deletePersonCommandParser.
close #41

### CODE REVIEW

File: src/main/java/seedu/sellsavvy/commons/core/LogsCenter.java [hunk 1]
```diff
@@ -75,11 +75,11 @@ private static void removeHandlers(Logger logger) {
     }
 
     /**
-     * Creates a logger named 'ab3', containing a {@code ConsoleHandler} and a {@code FileHandler}.
+     * Creates a logger named 'sellsavvy', containing a {@code ConsoleHandler} and a {@code FileHandler}.
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/commons/core/LogsCenter.java:78]: logger which I edited

File: src/main/java/seedu/sellsavvy/commons/core/LogsCenter.java [hunk 2]
```diff
@@ -75,11 +75,11 @@ private static void removeHandlers(Logger logger) {
...
      * Sets it as the {@code baseLogger}, to be used as the parent logger of all other loggers.
      */
     private static void setBaseLogger() {
-        baseLogger = Logger.getLogger("ab3");
+        baseLogger = Logger.getLogger("sellsavvy");
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/commons/core/LogsCenter.java:82]: ^

File: src/main/java/seedu/sellsavvy/logic/parser/DeletePersonCommandParser.java [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.personcommands.DeletePersonCommand;
-import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.commands.personcommands.DeletePersonCommand;
+import seedu.sellsavvy.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new DeleteCommand object
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/logic/parser/DeletePersonCommandParser.java:10]: Missed in previous refactoring

File: src/main/java/seedu/sellsavvy/logic/parser/EditPersonCommandParser.java [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
-import seedu.address.model.tag.Tag;
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.commands.personcommands.EditPersonCommand;
+import seedu.sellsavvy.logic.commands.personcommands.EditPersonCommand.EditPersonDescriptor;
+import seedu.sellsavvy.logic.parser.exceptions.ParseException;
+import seedu.sellsavvy.model.tag.Tag;
 
 /**
- * Parses input arguments and creates a new EditCommand object
+ * Parses input arguments and creates a new EditPersonCommand object
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/logic/parser/EditPersonCommandParser.java:23]: Missed in previous refactoring

File: src/main/java/seedu/sellsavvy/ui/HelpWindow.java [hunk 5]
```diff
@@ -8,14 +8,14 @@
...
-import seedu.address.commons.core.LogsCenter;
+import seedu.sellsavvy.commons.core.LogsCenter;
 
 /**
  * Controller for a help page
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://ay2425s1-cs2103t-f14a-2.github.io/tp/";
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/ui/HelpWindow.java:18]: changed the link for user guide

File: src/main/resources/view/MainWindow.fxml [hunk 6]
```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="SellSavvy" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```
Comment (FooChao) [inline src/main/resources/view/MainWindow.fxml:15]: changed title of gui

**Review by FooChao — COMMENTED** [review]
Things I changed which is not import/package statement

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommand.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+import seedu.sellsavvy.logic.commands.Command;
+import seedu.sellsavvy.logic.commands.CommandResult;
+import seedu.sellsavvy.logic.commands.exceptions.CommandException;
+import seedu.sellsavvy.model.Model;
+
+/**
+ * Adds an order under a specified person.
+ */
+public class AddOrderCommand extends Command {
```
Comment (kaixuan477) [inline src/main/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommand.java:11]: Might need to add test cases for this?

**Review by awdse22 — APPROVED** [review]

**Pull request merged**